e2e/cucumber.conf.js

```diff
@@ -13,7 +12,12 @@ exports.config = {
   specs: [
     './src/**/*.feature',
   ],
-  onPrepare() {
+  onPrepare: function () {
+
+    const {Given, Then, When} = require('cucumber');
+    global.Given = Given;
+    global.When = When;
+    global.Then = Then;
```

Comment:
this is not necessary

Reply:
Agree with you ...  i thought I want call those cucumber calls globally  so am gonna remove that code